content/03.methods.md

```diff
@@ -11,10 +11,14 @@
 2. Describe metadata statistics on PubMed central (number of articles, how many articles were processed
 
 ### Comparing Corpora
-1. Spacy to process text via - Lemmatization, removal of stop words
-2. Describe counting frequencies of each lemma
-3. Describe using chi-square test
-4. Describe how to calculate the likelihood and log odds ratio
+We used gensim [@raw:rehurek_lrec] (version 3.8.1) to perform the above preprocessing tasks on the bioRxiv and PubMed Central corpora.

```

Comment:
```suggestion
We used gensim [@raw:rehurek_lrec] (version 3.8.1) to preprocess the bioRxiv and PubMed Central corpora.
```
Is this correct? "the above preprocessing tasks" remains really confusing to me, since I don't think there's anything above this.

Reply:
Yeah that's correct. I was suppose to remove "the above" since the intended goal was for this sentence to come first.